.github/workflows/build.yml

```diff
@@ -53,12 +53,51 @@ jobs:
           pip install -U numpy
       - name: Build and install
         run: rm -rf .eggs && pip install -e .
-      - name: Run unittests and generate coverage report
+      - name: Run python unittests and generate coverage report
         run: |
           coverage run --branch --source mmdeploy -m pytest -rsE tests
           coverage xml
           coverage report -m
 
+  build_cpu_sdk:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          submodules: 'recursive'
+      - name: update
+        run: sudo apt update
+      - name: gcc-multilib
+        run: |
+          sudo apt install gcc-multilib g++-multilib wget libprotobuf-dev protobuf-compiler
+          sudo apt update
+          sudo apt install -y ffmpeg libsm6 libxext6 git ninja-build libglib2.0-0 libxrender-dev libc++1-9 libc++abi1-9
+          sudo add-apt-repository ppa:ignaciovizzo/opencv3-nonfree
+          sudo apt install libopencv-dev lcov wget
+          pkg-config --libs opencv
+      - name: Build and run SDK unit test without backend
```

Comment:
could we add the building of sdk demo as well?

Reply:
No, building demo means introducing  multiple backend.

Reply:
To improve test coverage,  end2end test would be low efficient.